maths/sum_of_digits.py

```diff
@@ -0,0 +1,18 @@
+def digit_sum(n: int) -> int:
+    """
+    Find the sum of digits of a number.
+
+    >>> digit_sum(12345)
+    15
+    >>> digit_sum(123)
+    6
+    """
+    res = 0
+    while n > 0:
+        res += n % 10
```

Comment:
Could we optimize using Python builtin https://docs.python.org/3/library/functions.html#divmod ?

Reply:
Yeah, divmod() method would be faster than mod operator.  But for small values, there is not much difference. Everytime, we mod a number with 10 until it becomes 0 will not have much effect on the running time I guess.

Reply:
```python
def digits(n):
        while n:
            n, digit = divmod(n, 10)
            yield digit
         
print(sum(digits(12345)))
```

Reply:
Yeah, that's the digit sum function using divmod()